### PR TITLE
Update object

### PR DESCRIPTION
**Pull request by maddygriz**
the nsclc object in the coercion vignette is too old to test this update but works with datasets from v2.0.
When we address #141 we can change the dataset to use this function or just update it to v2.2

### CODE REVIEW

**Comment by maddygriz** [comment]
resolves #155

File: R/NanoStringGeoMxSet-class.R [hunk 1]
```diff
@@ -29,6 +29,15 @@ function(object) {
     cat("\n")
 })
 
+setMethod("updateObject", signature = "NanoStringGeoMxSet",
```
Comment (NicoleEO) [inline R/NanoStringGeoMxSet-class.R:32]: Add roxygen comments or add to man page. Also, add importFrom BiocGenerics updateObject

File: vignettes/GeomxSet_coercions.Rmd [hunk 2]
```diff
@@ -197,6 +186,18 @@ colnames(protocolData(nsclc))[colnames(protocolData(nsclc)) == "StitchedReads"]
...
+
+nsclc@analyte <- "RNA"
+```
+
+```{r GeoMx dataset with coordinates}
+nsclc
+
+dim(nsclc)
+
+data.frame(nsclc@assayData$exprs[seq_len(5), seq_len(5)])
```
Comment (NicoleEO) [inline vignettes/GeomxSet_coercions.Rmd:198]: Use accessor

Comment (NicoleEO) [inline vignettes/GeomxSet_coercions.Rmd:198]: Use accessor

File: vignettes/GeomxSet_coercions.Rmd [hunk 3]
```diff
@@ -197,6 +186,18 @@ colnames(protocolData(nsclc))[colnames(protocolData(nsclc)) == "StitchedReads"]
 colnames(protocolData(nsclc))[colnames(protocolData(nsclc)) == "SequencingSaturation"] <- "Saturated (%)"
 
 protocolData(nsclc) <- protocolData(nsclc)[,-which(colnames(protocolData(nsclc)) %in% c("UID"))]
+
+nsclc@analyte <- "RNA"
```
Comment (NicoleEO) [inline vignettes/GeomxSet_coercions.Rmd:190]: Did you mean to updateObject(nsclc) here?

Comment (NicoleEO) [inline vignettes/GeomxSet_coercions.Rmd:190]: Did you mean to updateObject(nsclc) here?

**Review by NicoleEO — CHANGES_REQUESTED** [review]

File: vignettes/GeomxSet_coercions.Rmd (hunk 3 above)
Comment (maddygriz) [reply vignettes/GeomxSet_coercions.Rmd:190]: Nope. nsclc is a patch job of a geomxset object (that isn't technically valid for the last version) so it breaks the updateObject for reasons unrelated to this update. We really need to address it in it's own package which is where #141 comes into play but I don't think we need to do it for this release.

**Review by NicoleEO — APPROVED** [review]

**Pull request merged**